web/src/pages/Route/constants.ts

```diff
@@ -90,23 +90,87 @@ export const INIT_CHART = {
   hovered: {},
 };
 
-export const HASH_KEY_LIST = [
-  'remote_addr',
-  'host',
-  'uri',
-  'server_name',
-  'server_addr',
-  'request_uri',
-  'query_string',
-  'remote_port',
-  'hostname',
-  'arg_id',
-];
-
-export const HASH_ON_LIST = ['vars', 'header', 'cookie', 'consumer'];
-
 export const AUTH_LIST = ['basic-auth', 'jwt-auth', 'key-auth'];
 
+export const HEADER_LIST = [
+  "Accept",
+  "Accept-Charset",
+  "Accept-Encoding",
+  "Accept-Language",
+  "Accept-Ranges",
+  "Access-Control-Allow-Credentials",
+  "Access-Control-Allow-Origin",
+  "Access-Control-Allow-Methods",
+  "Access-Control-Allow-Headers",
+  "Access-Control-Max-Age",
+  "Access-Control-Expose-Headers",
+  "Access-Control-Request-Method",
+  "Access-Control-Request-Headers",
+  "Age",
+  "Allow",
+  "Authorization",
+  "Cache-Control",
+  "Connection",
+  "Content-Encoding",
+  "Content-Language",
+  "Content-Length",
+  "Content-Location",
+  "Content-Range",
+  "Content-Security-Policy",
+  "Content-Type",
+  "Cookie",
+  "DNT",
+  "Date",
+  "ETag",
+  "Expect",
+  "Expires",
+  "From",
+  "Host",
+  "If-Match",
+  "If-Modified-Since",
+  "If-None-Match",
+  "If-Range",
+  "If-Unmodified-Since",
+  "Last-Event-ID",
+  "Last-Modified",
+  "Link",
+  "Location",
+  "Max-Forwards",
+  "Negotiate",
+  "Origin",
+  "Pragma",
+  "Proxy-Authenticate",
+  "Proxy-Authorization",
+  "Range",
+  "Referer",
+  "Retry-After",
+  "Sec-Websocket-Extensions",
+  "Sec-Websocket-Key",
+  "Sec-Websocket-Origin",
+  "Sec-Websocket-Protocol",
+  "Sec-Websocket-Version",
+  "Server",
```

Comment:
hi @qian0817, I noticed that there are some response header fields in this list. it would be better to only keep request header fields here. for more information, you can refer to https://en.wikipedia.org/wiki/List_of_HTTP_header_fields `Standard request fields` and `Common non-standard request fields`, thanks